### PR TITLE
[refactor] remove trailing whitespace

### PR DESCRIPTION
**Pull request by pieleric**
Now that we have a linter complaining about it, it's better to fix the current files.

Essentially, run:
```
find . -name '*.py' -exec sed -i 's/[ \t]*$//' '{}' ';'
autopep8 --select W291,W292,W293,W391 --in-place ./**.py
```

### CODE REVIEW

**Review by patrickcleeve2 — APPROVED** [review]

File: plugins/cl_tcorr_acq.py
Comment (patrickcleeve2) [inline plugins/cl_tcorr_acq.py:1]: Linter is still failing trailing whitespace on this file. 

```bash
plugins/cl_tcorr_acq.py:50:113: W291 trailing whitespace
plugins/cl_tcorr_acq.py:461:76: W291 trailing whitespace
````

Comment (patrickcleeve2) [reply plugins/cl_tcorr_acq.py:1]: https://github.com/delmic/odemis/blob/e7cd5e2ae8362525371724ac4ffac194350c36e7/plugins/cl_tcorr_acq.py#L50

Comment (patrickcleeve2) [reply plugins/cl_tcorr_acq.py:1]: https://github.com/delmic/odemis/blob/e7cd5e2ae8362525371724ac4ffac194350c36e7/plugins/cl_tcorr_acq.py#L461

**Review by patrickcleeve2 — CHANGES_REQUESTED** [review]

File: plugins/cl_tcorr_acq.py
Comment (pieleric) [reply plugins/cl_tcorr_acq.py:1]: Indeed, this file has windows line endings (CRLF), which confuses sed, and it's in a docstring, which autopep8 doesn't correct.
I've just fixed that.

**Review by tepals — APPROVED** [review]

**Pull request merged**